servers/register_server_types.cpp

```diff
@@ -264,6 +264,7 @@ void register_server_types() {
 	GDREGISTER_ABSTRACT_CLASS(NavigationServer2D);
 	GDREGISTER_CLASS(NavigationPathQueryParameters2D);
 	GDREGISTER_CLASS(NavigationPathQueryResult2D);
+	GDREGISTER_CLASS(GodotNavigationServer2D);
```

Comment:
This doesn't look correct, this is not a public class to register, it's an implementation of the general navigation server, just like `GodotPhysicsServer2D`, it should at least be internal